platform/conf/conf.go

```diff
@@ -135,6 +149,32 @@ func Script(data string) *UserData {
 	}
 }
 
+func decompressIfGzipped(data []byte) []byte {
```

Comment:
We could add a hint that these functions are also implemented in coreos-cloudinit if I'm not mistaken and should be kept in sync?

Reply:
Also, if we have any duplication in `platform/conf/multipart.go`, that could also benefit from pointers. Even if it's not a duplication I think it's nice to link the similar code.

Reply:
Also, they don't really need to be in sync. As long as they serve their function, they can diverge. I think we should at some point create that utility package.